src/explore-education-statistics-admin/src/services/publicationService.ts

```diff
@@ -47,28 +51,36 @@ export interface MyPublication {
   externalMethodology?: ExternalMethodology;
   topicId: string;
   themeId: string;
-  contact: PublicationContactDetails;
-  permissions: {
-    canAdoptMethodologies: boolean;
-    canCreateReleases: boolean;
-    canUpdatePublication: boolean;
-    canUpdatePublicationTitle: boolean;
-    canUpdatePublicationSupersededBy: boolean;
-    canCreateMethodologies: boolean;
-    canManageExternalMethodology: boolean;
-  };
+  contact: Contact;
+  permissions: PublicationPermissions;
   supersededById?: string;
   isSuperseded?: boolean;
 }
 
+export interface PublicationPermissions {
+  canAdoptMethodologies: boolean;
+  canCreateReleases: boolean;
+  canUpdatePublication: boolean;
+  canUpdatePublicationTitle: boolean;
+  canUpdatePublicationSupersededBy: boolean;
+  canCreateMethodologies: boolean;
+  canManageExternalMethodology: boolean;
+}
+
+export interface PublicationWithPermissions extends Publication {
+  permissions: PublicationPermissions;
+}
+
 export interface Publication {
   id: string;
   title: string;
   summary: string;
   slug: string;
-  contact: PublicationContactDetails;
   theme: IdTitlePair;
   topic: IdTitlePair;
+  supersededById?: string;
+  isSuperseded?: boolean;
```

Comment:
Could this just be inferred from the existence of a non-null `supersededById`?  Not a biggie, probably makes it more readable...

Reply:
Unfortunately not. For `isSuperseded` to be true, the `supersededById` publication must also have a live release.

Reply:
I might be tempted to rename `isSuperseded` to something more explicit then, like `isSupersededByLivePublication` or put a code comment next to `isSuperseded` to explain the situation - cos for me personally I'd be confused in a scenario whereby `isSuperseded` is false but `supersededById` is not null :)

Reply:
I've added a comment in both the backend and frontend that should make this clearer. With hindsight, I think I would have named it `isSupersededByLivePublication`, as that clears up the ambiguity. But I'm inclined not to rename it now as that means invalidating all publications' publication.json. And since this work is likely to be deployed alongside the read replica work, we probably want as few things to worry about in that deploy as possible.

But still happy to rename it if you'd prefer - or I could create a ticket to rename it at a later date.

Reply:
Just to chime in here, but a more terse suggestion would be `isSupersededByLive`